Makefile

```diff
@@ -131,16 +133,23 @@ clean-ui:
 .PHONY: clean
 clean: clean-ui clean-backend ## Clean all build artifacts
 
-bin/golangci-lint: bin/golangci-lint-${GOLANGCI_VERSION}
-	@ln -sf golangci-lint-${GOLANGCI_VERSION} bin/golangci-lint
-bin/golangci-lint-${GOLANGCI_VERSION}:
-	@mkdir -p bin
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | bash -s -- -b ./bin/ v${GOLANGCI_VERSION}
+$(BIN_DIR):
+	@mkdir -p $(BIN_DIR)
```

Comment:
Need to add this as an order only dependency to all the `bin/vmclarity-...` targets above.

Reply:
Wont fix in this PR as those are out of scope for this fix.